ml/src/main/scala/ai/h2o/sparkling/ml/params/H2OGridSearchParams.scala

```diff
@@ -20,31 +20,23 @@ import java.util
 
 import ai.h2o.sparkling.ml.algos.{H2OGridSearch, H2OSupervisedAlgorithm}
 import ai.h2o.sparkling.ml.internals.H2OMetric
-import hex.grid.HyperSpaceSearchCriteria
-import hex.{Model, ScoreKeeper}
+import hex.Model
```

Comment:
NIT: extra import?

Reply:
thanks!

Reply:
In fact, the import is required by [the line 65](https://github.com/h2oai/sparkling-water/pull/2129/files/175f47c5ca617d0b60e9f29a0333a1667d3cc2a0#diff-7419246e33dc368bbd698a3f802b79c2R65)

Reply:
You're right, sorry